api/src/main/java/org/openmrs/module/pihcore/config/Components.java

```diff
@@ -18,14 +18,19 @@
         public static final String ONCOLOGY = "oncology";  // program
         public static final String NCD = "ncd";  // program
         public static final String HIV = "hiv";  // program. Needs HIVProgramBundle and Haiti HIV MDS package
```

Comment:
Let's remove the "program" part of this comment here, which was outdated before but now is particularly confusing with the "hivProgram" component as well below.